platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/components/PostHashCollectorWiring.java

```diff
@@ -1,70 +0,0 @@
-/*
- * Copyright (C) 2024 Hedera Hashgraph, LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.swirlds.platform.wiring.components;
-
-import com.swirlds.common.wiring.schedulers.TaskScheduler;
-import com.swirlds.common.wiring.wires.input.BindableInputWire;
-import com.swirlds.common.wiring.wires.input.InputWire;
-import com.swirlds.common.wiring.wires.output.OutputWire;
-import com.swirlds.platform.event.GossipEvent;
-import edu.umd.cs.findbugs.annotations.NonNull;
-import java.util.function.LongSupplier;
-
-/**
- * Wiring object that allows for the staging of events that have been hashed, but haven't been passed further down the
```

Comment:
This big ol explanation for why we need a `PostHashCollector` needs to find a new home, since I don't think any other part of the code explains the purpose

Reply:
It has been re-homed into the PassThroughWiring class and adapted slightly since the PassThroughWiring is more general of a use case.  Please review my adaptation and suggest changes to your satisfaction.

Reply:
LGTM